config/dynamic-scaling-configuration.yaml

```diff
@@ -1,7 +1,17 @@
+#This configuration file contains information about each Kafka instance types, dynamic scaling configuration
+#
+#The following properties must be defined for each Kafka instance type:
+#   - reserved_streaming_units: the number of streaming unit reservations to be made on each data plane cluster
+#
+#The following properties must be defined for each instance types compute node configuration:
+#   - max_compute_nodes: the maximum number of nodes needed to create an appropriate machine pool with the configuration needed by the autoscaler
+
 ---
 developer:
+  reserved_streaming_units: 3 
   compute_nodes_config:
     max_compute_nodes: 3
 standard:
+  reserved_streaming_units: 10 
```

Comment:
wdyt of changing this to 1 or maybe 2 by default? I don't think we need to reserve 10 for local dev. Same with developer. We should try and minimise the amount of nodes to scale up to for local dev.

cc @machi1990

Reply:
This make sense, thank you for catching this.

Also considering the max_nodes is `3` that's won't be enough to fit `10` streaming units.

Reply:
Thanks! See that it's updated now, closing as resolved :)